### PR TITLE
Add Admin / Conservation manager

### PR DESCRIPTION
**Pull request by bbarber**
Pull Request Checklist

- [x] Did you pull latest and merge `develop` (or `master`) into your branch?
- [x] Did you add tests?
- [ ] Did you run the front-end tests (if applicable)?
- [x] Did you run the back-end tests (if applicable)?
- [ ] Did you include screenshots or a demo video (if applicable)?
- [x] Did you include a descriptive title and description with your PR?
- [x] Did you perform a self-review before assigning reviewers?

### CODE REVIEW

File: src/API/WesternStatesWater.WestDaat.Tests.IntegrationTests/Conservation/ApplicationIntegrationTests.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+namespace WesternStatesWater.WestDaat.Tests.IntegrationTests.Conservation;
+
+[TestClass]
+public class ApplicationIntegrationTests : IntegrationTestBase
+{
+}
```
Comment (bbarber) [inline src/API/WesternStatesWater.WestDaat.Tests.IntegrationTests/Conservation/ApplicationIntegrationTests.cs:6]: Intentionally left empty until proxy pattern is established

**Review by tarrball — CHANGES_REQUESTED** [review]

File: src/API/WesternStatesWater.WestDaat.Contracts.Client/IApplicationManager.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using WesternStatesWater.WestDaat.Common;
+
+namespace WesternStatesWater.WestDaat.Contracts.Client;
+
+public interface IApplicationManager : IServiceContractBase
+{
+    Task<ResponseBase> Load(RequestBase request);
+    
+    Task<ResponseBase> Store(RequestBase request);
```
[the remark this answers is not in the document]
Comment (bbarber) [reply src/API/WesternStatesWater.WestDaat.Contracts.Client/IApplicationManager.cs:13]: Thoughts on this dir / namespace?
image

[the remark this answers is not in the document]
Comment (bbarber) [reply src/API/WesternStatesWater.WestDaat.Contracts.Client/IApplicationManager.cs]: Pushed

File: src/API/WesternStatesWater.WestDaat.Client.Functions/Program.cs [hunk 3]
```diff
@@ -45,7 +45,9 @@
         services.AddScoped(a => configuration.GetBlobStorageConfiguration());
         services.AddScoped(a => configuration.GetPerformanceConfiguration());
 
+        services.AddScoped<IApplicationManager, ConservationManager>();
         services.AddScoped<INotificationManager, NotificationManager>();
+        services.AddScoped<IUserManager, AdminManager>();
```
[the remark this answers is not in the document]
Comment (bbarber) [reply src/API/WesternStatesWater.WestDaat.Client.Functions/Program.cs]: Agree - Updated to use `AddTransient`

**Review by tarrball — APPROVED** [review]

**Pull request merged**